gradle.properties

```diff
@@ -17,6 +17,7 @@ android.enableR8.fullMode=true
 android.nonFinalResIds=true
 android.nonTransitiveRClass=true
 android.useAndroidX=true
+android.experimental.androidTest.numManagedDeviceShards=3
```

Comment:
anything to note about how you figured out 3 shards? e.g. if we wanted to change this number in the future, how would we figure out a good number to change it to?